### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by hanssto**
Extract general points, and tie the two pieces more together.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,34 @@
...
+
+* If you have any questions, please don't hesitate to contact us.
+
+## Practical case: Deployable development environment
+
+### Premise
 
 Provided are two simplified parts of the same application environment: A database dump and an API service. Your task is to automate setting up the development environment in a reliable and testable manner using "infrastructure as code" principles.
 
 The goal is to end up with a limited set of commands that would install the different environments and run them using containers. You can use any software that you find suitable for the task. The code should come with instructions on how to run it and deploy it to arbitrary targets; whether it is deployed locally, towards physical machines, or towards virtual nodes in the cloud.
```
Comment (nkmadusanka) [inline README.md:29]: Shall we make it clear that we expect to deploy into a production environment? I mean use the words like, production/staging/QA, etc.?

Comment (nkmadusanka) [reply README.md:29]: I totally missed this point in my solution

Comment (hanssto) [reply README.md:29]: We are asking people to make multiple development environments, not prod, staging, QA, really. It'd be natural to ask how to extend things to prod in an interview though. If we want to go another way here, we need to rework the text a bit. Should we?

**Review by luc-m — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (nkmadusanka) [reply README.md:29]: Oh, ok then my initial understanding was not incorrect then, leave it then.

Current is fine.

**Review by nkmadusanka — APPROVED** [review]
LGTM

**Pull request merged**